build/build.ps1

```diff
@@ -87,8 +82,52 @@ function LocateVisualStudio {
     [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
     Invoke-WebRequest "http://github.com/Microsoft/vswhere/releases/download/$vswhereVersion/vswhere.exe" -OutFile $vswhereExe
   }
+  
+  return $vsWhereExe
+}
+
+function GetVSIXExpInstallerExe{
+  $vsixExpInstallerVersion = GetVersion("VSIXExpInstallerVersion")
+  $vsixExpInstalleDir = Join-Path $ToolsRoot "VSIXExpInstaller\$vsixExpInstallerVersion"
+  $vsixExpInstalleExe = Join-Path $vsixExpInstalleDir "tools\VSIXExpInstaller.exe"
+  
+  if (!(Test-Path $vsixExpInstalleExe)) {
+    Create-Directory $vsixExpInstalleDir
+    Write-Host "Downloading VSIXExpInstaller"
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    Invoke-WebRequest "https://dotnet.myget.org/F/roslyn-tools/api/v2/package/RoslynTools.VSIXExpInstaller/$vsixExpInstallerVersion" -OutFile RoslynTools.VSIXExpInstaller.zip
+    Expand-Archive .\RoslynTools.VSIXExpInstaller.zip -DestinationPath $vsixExpInstalleDir
+  }
+  
+  return $vsixExpInstalleExe
+}
+
+function GetTRXUnitExe{
+  $trxUnitVersion = GetVersion("TRXUnitVersion")
+  $trxUnitDir = Join-Path $ToolsRoot "TRXUnit\$trxUnitVersion"
+  $trxUnitExe = Join-Path $trxUnitDir "tools\TRXunit.exe"
+  if (!(Test-Path $trxUnitExe)) {
+    Create-Directory $trxUnitDir
+    Write-Host "Downloading TRXUnit"
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    Invoke-WebRequest "https://dotnet.myget.org/F/roslyn-tools/api/v2/package/TRXunit/$trxUnitVersion" -OutFile TRXUnit.zip
+    Expand-Archive .\TRXUnit.zip -DestinationPath $trxUnitDir
+  }
+  
+  return $trxUnitExe
+}
+
+function InstallVSIX([string] $vsixExpInstalleExe, [string] $rootsuffix, [string] $vsInstallDir, [string] $pathToVSIX){
+  & $vsixExpInstalleExe /rootSuffix:$rootsuffix /vsInstallDir:$vsInstallDir $pathToVSIX
```

Comment:
Does this work if `rootSuffix` is empty?

Reply:
Yes